Makefile

```diff
@@ -308,6 +308,7 @@ $(BUILD_DIR)/image/%/$(DUMMY): Makefile $(BUILD_DIR)/image/%/payload $(BUILD_DIR
 	$(DBUILD) -t $(DOCKER_NS)/fabric-$(TARGET) $(@D)
 	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET):$(DOCKER_TAG)
 	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET):$(ARCH)-latest
+	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET):$(ARCH)-$(BASEIMAGE_RELEASE)
```

Comment:
I'm trying to understand why this one is BASEIMAGE_RELEASE rather than BASE_VERSION.

Reply:
You're right, the right thing to do here would be to hard code the tag like we do in release-2.x branches and tag the image proper in the `ccenv` target. I'll make that change